.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Test
+on:
+  push:
+    branches: [main, master, develop]
+    tags: ["v*"]
+    paths:
+      - ".github/workflows/**"
+      - "packages/basemap/**"
+  pull_request:
+    branches: [main, master, develop]
+    paths:
+      - ".github/workflows/**"
+      - "packages/basemap/**"
+  workflow_dispatch: # Allows manual workflow runs
+env:
+  PKGDIR: "packages/basemap"
+  CIBW_BUILD_VERBOSITY: 1
+  PYTHONUNBUFFERED: "1"
+  PYTHONWARNINGS: "ignore:DEPRECATION"
+  GEOS_VERSION: "3.6.5"
```

Comment:
I find nice that we set `GEOS_VERSION` as environment variable here, but then we need to use this variable later when calling `GeosLibrary.build` instead of having "3.6.5" hardcoded.

Reply:
Agreed!